### DISCUSSION TITLE
How to exclude index.html from linting by @angular-eslint/template?

### DISCUSSION THREAD

**Discussion by belyan** [angular-eslint/angular-eslint#1213] [Q&A] [answered]
If I add "index.html" to excluded files, when I get this error:
```
/Users/.../Projects/.../src/index.html
  1:1  error  Parsing error: Unexpected token <
```

My `eslintrc.json` file looks like:
```json
{
  "root": true,
  "ignorePatterns": ["projects/**/*"],
  "overrides": [
    {
      "files": ["*.ts"],
      "parserOptions": {
        "project": ["tsconfig.json"],
        "createDefaultProgram": true
      },
      "extends": ["plugin:@typescript-eslint/recommended", "plugin:@angular-eslint/recommended"],
      "rules": {}
    },
    {
      "files": ["*.html"],
      "excludedFiles": ["src/index.html"],
      "extends": ["plugin:@angular-eslint/template/recommended"],
      "rules": {}
    }
  ]
}
```

If I add "index.html" to `ignorePatterns` then error will gone, but, as for me, it is looks not logical to use global pattern to exclude one HTML file.

**Comment by JamesHenry** [comment, marked as answer]
`excludedFiles` excludes the file from consideration for that override block. So what you are doing is instructing ESLint _not_ to use the Angular HTML parsing logic and fallback to its default (JavaScript) which of course blows up on HTML.

`ignorePatterns` or using a `.eslintignore` file is the way to go

**Reply by belyan** [reply]
Okay, I understood my mistake. Thanks!
